src/StackAdmin/Azs.Storage.Admin/Module/Azs.Storage.Admin/Azs.Storage.Admin.psd1

```diff
@@ -116,7 +116,7 @@ PrivateData = @{
         # IconUri = ''
 
         # ReleaseNotes of this module
-        ReleaseNotes = '* SDK for AzureStack SRP in 2019-08-08-preview'
+        ReleaseNotes = '* SDK for AzureStack SRP in 2019-08-08-preview. The new version removes the following commands: Get-AzsBlob*, Get-AzsQueue*, Get-AzsTable*, Get-AzsStorageShare* and *StorageContainerMigration. Meanwhile, it adds new commands like *AzsStorageSettings to do operation on settings, e.g. retention days. In the new version, the *FARM* concept is removed. The usage of powershell commands are almost the same except you do not need to specify *FARM*. For example, you need to call *Get-AzsStorageAccount -FarmName XXXXXX* in the previous versions and just *Get-AzsStorageAccount* in current version.'
```

Comment:
can you please loop in your PM to sign off on this? I believe the release notes could be a more detailed, or you should add a migration guide like this one and link it in the release notes with a aka.ms link
https://github.com/Azure/azure-powershell/blob/AzureRM/documentation/migration-guides/Stack/azurestack.1.7.1.md

We absolutely need Old usage vs New Usage so that the breaking changes are addressed well. List the deprecated cmdlets in full form and the suggested replacements for the deprecated cmdlets